app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :post
```

Comment:
- [ ] Great job so far @yemidada There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink:
- [ ] From the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), you can see that a comment belongs to both a user and a post but you have only established the association to a post and not to a user. So kindly fix that :+1: